org.eclipse.jdt.debug.ui/ui/org/eclipse/jdt/internal/debug/ui/ErrorDialogWithToggle.java

```diff
@@ -43,25 +47,37 @@ public class ErrorDialogWithToggle extends ErrorDialog {
 	/**
 	 * The message displayed to the user, with the toggle button
 	 */
-	private String fToggleMessage= null;
+	private String fToggleMessage1 = null;
```

Comment:
Style: please don't initialize fields to default values, java does it automatically, but during debugging you will jump from field to field just because of this meaningless instructions here.

Reply:
will remove default initialization